### PR TITLE
SD-62: styling for login DONE ! :D

### PR DESCRIPTION
**Pull request by kkatlopez**

### CODE REVIEW

File: client/src/components/UserLogin.js [hunk 1]
```diff
@@ -78,48 +76,45 @@ class UserLanding extends Component {
...
+                value={this.state.pass} onChange={this.changePass} isInvalid={!this.state.isubmittable}/>
+
+                <Form.Control.Feedback type="invalid">
+                  Either your username or password is incorrect.
+                </Form.Control.Feedback>
+              </FloatingLabel>
+            </Form.Group>
+            
+            <Form.Group className="mt-4 d-flex justify-content-center">
+              <Button onClick={this.confirmCreds} className={"green-button " + (this.state.submittable ? "" : "disabled")} disabled={!this.state.submittable}>Sign in</Button>
```
Comment (matthewyoungbar) [inline client/src/components/UserLogin.js:113]: This button is unselectable if the browser auto-fills the username and password, which is rather annoying.
It is also still unclickable on iOS (both Safari and iOS Chrome).

**Review by matthewyoungbar — CHANGES_REQUESTED** [review]

File: client/src/components/UserLogin.js (hunk 1 above)
Comment (kkatlopez) [reply client/src/components/UserLogin.js:113]: i think were fine for the autofill thing... but not sure what to do about the unclickable on iOS, did gwyneth's branch on SD-71 fix that? it hasn't been merged to main

Comment (matthewyoungbar) [reply client/src/components/UserLogin.js:113]: No I don't think we got it to work on that branch either

**Review by gwynethyuen — APPROVED** [review]
we can fix the ios/touch thing in SD-71

**Review by matthewyoungbar — APPROVED** [review]

**Pull request merged**